test/gtest/ucp/test_ucp_am.cc

```diff
@@ -863,6 +863,55 @@ UCS_TEST_P(test_ucp_am_nbx_send_flag, eager, "RNDV_THRESH=128")
 UCP_INSTANTIATE_TEST_CASE(test_ucp_am_nbx_send_flag)
 
 
+class test_ucp_am_nbx_short_slow_path : public test_ucp_am_nbx {
```

Comment:
You can use this class instead: `test_ucp_am_nbx_reply`

Reply:
I thought to use it but in this case the test will run with more variants... I thought to save time but also can change .

Reply:
It will add only `prereg` variant, which I will move from base class to inherited classes.

Reply:
Yes. will run 8 tests for each TL instead of 4.
I will take your advice unless someone else disagree.